app/model/Group.php

```
@@ -0,0 +1,238 @@
+            $this->addUsersToGroup($row->id, $data['users']);
+        }
+
+        return $row;
+    }
+
+    /**
+     * @param array $data
+     */
+    public function updateGroup($data)
+    {
+        $id = $data['id'];
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)